docs/source/datasets.py

```diff
@@ -46,10 +46,67 @@
 }
 
 
+def sort_schemas(schemas: Tuple[str, DatasetSchema]) -> List[Union[str, DatasetSchema]]:
+    """Sort datasets (schemas) alphabetically.
+
+    Args:
+        schemas: A set containing a pair of schemas id (name) and DatasetSchema instance.
```

Comment:
According to the type of the argument `set` should be called `tuple` to avoid confusion.

Reply:
A list of tuples, to be exact.

Reply:
:+1: